src/main/java/org/kohsuke/github/GHPersonSet.java

```diff
@@ -10,6 +10,8 @@
  * @author Kohsuke Kawaguchi
  */
 public class GHPersonSet<T extends GHPerson> extends HashSet<T> {
+    private static final long serialVersionUID = 1L;
+ 
```

Comment:
For newly created classes the preference is to force the value to `1L` so that we know it was always set from the start